src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,39 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, onTabSelected, selectedTabId }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            className={`${
+              tab.id === selectedTabId
+              || (!tabs.find(tabWithTheID => tabWithTheID.id === selectedTabId)
+              && tab.id === tabs[0].id)
+                ? 'is-active'
+                : ''
+            }`}
+            data-cy="Tab"
+            key={tab.id}
+          >
+            <a
+              href={`#${tab.id}`}
+              data-cy="TabLink"
+              onClick={() => {
+                if (tab.id !== selectedTabId) {
```

Comment:
It is better to move this check to a callback, which is passed to the component
the component should be as simple as possible

Reply:
![image](https://github.com/mate-academy/react_tabs-js/assets/55620816/ac594d13-c63c-42c5-8b17-a91284ed0a80)